src/settings.c

```diff
@@ -724,6 +724,7 @@ void load_settings(char *cmdline_config_path)
                 r->history_ignore = ini_get_bool(cur_section, "history_ignore", r->history_ignore);
                 r->match_transient = ini_get_bool(cur_section, "match_transient", r->match_transient);
                 r->set_transient = ini_get_bool(cur_section, "set_transient", r->set_transient);
+                r->desktop_entry = ini_get_string(cur_section, "desktop-entry", r->desktop_entry);
```

Comment:
We're using underscores for separating words in all other rules. Maybe it's best to keep it consistent and do the same here.